sdcm/utils/es_queries.py

```diff
@@ -35,12 +37,16 @@ def filter_test_details(self):
         test_details += ' AND test_details.test_name: {}'.format(self.test_name.replace(":", r"\:"))
         return test_details
 
+    def filter_test_for_lastyear(self):
+        year_ago = (datetime.today() - timedelta(days=365)).date().strftime("%Y%m%d")
+        return f"versions.scylla-server.date:{{{year_ago} TO *}}"
```

Comment:
is `versions.scylla-server.date` index as data ? or as string in elastic search ?
I think it would be quicker if we index it as a date.

Reply:
yes, it is indexed as text not as date. I think you right. It will better to reindex it as date, but this could affect on all test results